tests/EmitTest.elm

```diff
@@ -163,6 +163,22 @@ javascript =
                       )
                     ]
                 )
+            , describe "List"
+                (List.map runTest
```

Comment:
Let's write a comment here that this `[...]` format will eventually have to change to something like `List$fromArray([...])` when we add `(::)`, `(++)`, etc - this piggybacking on platform's collections has to end somewhere and get converted to the actual Elm values